File: DevExpress-Demo/CS/WindowsUIDemo.Wpf/Modules/HamburgerMenuDemo/CompactModeModel.cs
using System.Collections.Generic;
using System.Linq;

namespace WindowsUIDemo {
    public class CompactModeModel {
        public IEnumerable<Message> Data { get { return MailItems.Messages; } }    

        public Message FocusedRow { get; set; }

        public CompactModeModel() {
            FocusedRow = Data.First();
        }
    } 
}

File: DevExpress-Demo/VB/WindowsUIDemo.Wpf/Modules/HamburgerMenuDemo/CompactModeModel.vb
Imports System.Collections.Generic
Imports System.Linq

Namespace WindowsUIDemo

    Public Class CompactModeModel

        Public ReadOnly Property Data As IEnumerable(Of Message)
            Get
                Return Messages
            End Get
        End Property

        Public Property FocusedRow As Message

        Public Sub New()
            FocusedRow = Data.First()
        End Sub
    End Class
End Namespace
